uftrace.c

```diff
@@ -558,6 +558,16 @@ static int parse_option(struct opts *opts, int key, char *arg)
 		opts->hide = opt_add_string(opts->hide, arg);
 		break;
 
+	case 'L':
```

Comment:
You can add `opts->no_sched = true` to suppress showing too many scheduling events just like https://github.com/namhyung/uftrace/commit/55ab685b8e05deb6d3dddcd565a7fc8deee3bc0d did.